paddle/c/scope.cc

```diff
@@ -0,0 +1,32 @@
+#include "scope.h"
```

Comment:
Whether uses C-API depends on is there any other languages need invoke Paddle C++ Core or not.

I am not sure only Python API is enough or not. At least there are several needs for us to give a C-API.

* Android/IOS API. Give a Java/Objective-C wrapper for Paddle.
* Go API. Needs for our Parameter Server.